docs/reference/sources_yaml.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_label: sources.yaml
+sidebar_position: 110
+title: Reference for sources.yaml
+description: Reference for sources.yaml
+keywords: [kubewarden, kubernetes, sources.yaml]
+doc-persona: [kubewarden-operator, kubewarden-integrator]
+doc-type: [reference]
+doc-topic: [operator-manual, sources.yaml]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/reference/sources_yaml"/>
+</head>
+
+## The `sources.yaml` file
+
+You can tune the push-pull behavior of `kwctl` and `policy-server` using the `sources.yaml` file.
```

Comment:
Not sure, maybe it's worth it to move the paragraph about how to consume the `sources.yaml` out of the reference page and into the howto.

Reply:
I think it's ok there. For when you land on the ref page and wonder what you can do with the material there. Also, ok on a how-to as well. We can change in the light of experience.